specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json

```diff
@@ -1043,6 +1043,40 @@
         }
       }
     },
+    "WorkspaceRepositoryConfiguration": {
+      "description": "Git integration settings",
+      "type": "object",
+      "properties": {
+        "accountName": {
+          "description": "Account name",
+          "type": "string"
+        },
+        "collaborationBranch": {
+          "description": "Collaboration branch",
+          "type": "string"
+        },
+        "hostName": {
```

Comment:
just for better readability (no functional impact) consider ordering these from largest scope to smallest - if I understand correctly that would be hostName, accountName, projectName, type, repositoryName, collaborationBranch, rootFolder or whatever you feel is the correct hierachy

Reply:
Will update the order and type must be first to choose.
type, hostName, accountName, projectName, repositoryName, collaborationBranch, rootFolder